src/app/data/pontusx-account-names.ts

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios'
+import { useQuery } from '@tanstack/react-query'
+import { Layer } from '../../oasis-nexus/api'
+import { Network } from '../../types/network'
+import { hasTextMatch } from '../components/HighlightedText/text-matching'
+import { AccountData, AccountMetadata, AccountMap, AccountMetadataInfo } from './named-accounts'
+
+const DATA_SOURCE_URL = 'https://raw.githubusercontent.com/deltaDAO/mvg-portal/main/pontusxAddresses.json'
+
+const getPontusXAccountsMetadata = () =>
+  new Promise<AccountData>((resolve, reject) => {
+    axios.get(DATA_SOURCE_URL).then(response => {
+      if (response.status !== 200) reject("Couldn't load names")
+      if (!response.data) reject("Couldn't load names")
```

Comment:
can be modernized with async await

Reply:
Yes, I have already done that with the Oasis counterpart, just had to step out a little bit..

Reply:
replaced with aync await

Reply:
replaced with async await